source/features/clean-checks-list.css

```diff
@@ -0,0 +1,34 @@
+/* Bringing running checks and failed checks to the top of the list */
+/* .octicon-check targets "successful" checks */
+/* .octicon-x targets "failed" checks */
+/* .octicon-skip targets "skipped" checks */
+/* .octicon-square-fill targets "neutral" checks */
+/* .text-italic targets "running" or "waiting" checks */
+
+/* Required to allow sorting */
+.merge-status-list {
+	display: flex;
+	flex-direction: column;
+}
+
+/* Push running checks below failed checks */
+.merge-status-item:has(.text-italic) {
+	order: 1;
+}
+
+/* Push successful/skipped/neutral checks below running checks */
+.merge-status-item:has(.octicon-check, .octicon-skip, .octicon-square-fill) {
+	order: 2;
+}
+
+/* Dim successful/skipped/neutral checks */
+.merge-status-list:has(.octicon-x, .text-italic) /* Only if any checks are failing or in progress */
+.merge-status-item:has(.octicon-check, .octicon-skip, .octicon-square-fill) {
```

Comment:
Maybe undim on hover:

```suggestion
.merge-status-item:has(.octicon-check, .octicon-skip, .octicon-square-fill):not(:hover) {
```

Reply:
I kinda wonder if we need to dim them at all. 🤔 Failing checks are already at the top.

## Dim

<img width="514" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/181234947-bbf01fc7-8f41-4352-9615-60a6fcdcdf61.png">

## No dim

<img width="514" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/1402241/181234969-5e4707a6-ce61-424a-81ee-887ca23c6636.png">

Reply:
Makes sense. `dim-bots` needs to work this way because stuff can be in the middle, while this just reorders stuff.

Reply:
On second thought, this looks super clean:

<img width="528" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/181428564-b7251105-2a9a-45a7-aab2-6e8176eaa197.png">